app/assets/stylesheets/application.scss

```diff
@@ -33,7 +33,7 @@
 }
 
 .dashed {
-  border: 2px dashed #808080;
+  border: 2px dashed #D3D3D3;
```

Comment:
Color literals like `#D3D3D3` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#D3D3D3` should be written as `#d3d3d3`